### PR TITLE
Add license of PKI.js

### PR DESCRIPTION
**Pull request by kweisamx**
ref: https://github.com/PeculiarVentures/PKI.js/blob/master/LICENSE, https://github.com/PeculiarVentures/PKI.js#license

### CODE REVIEW

File: ajax/libs/PKI.js/package.json [hunk 1]
```diff
@@ -30,5 +30,6 @@
         ]
       }
     ]
-  }
+  },
+  "license": "MIT"
```
Comment (pvnr0082t) [inline ajax/libs/PKI.js/package.json]: It seems like it's `BSD-3-Clause`. 
ref: https://github.com/PeculiarVentures/PKI.js/blob/master/LICENSE, https://github.com/PeculiarVentures/PKI.js#license

File: ajax/libs/PKI.js/package.json [hunk 2]
```diff
@@ -30,5 +30,6 @@
         ]
       }
     ]
-  }
+  },
+  "license": "BSD-3-clause"
```
Comment (sufuf3) [inline ajax/libs/PKI.js/package.json]: `BSD-3-Clause`
Please use upper case. c -> C
Let it as same with https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L49

Comment (kweisamx) [inline ajax/libs/PKI.js/package.json]: thank !

**Comment by PeterDaveHello** [comment]
@kweisamx I think you can try to send a pull request to update its repository url.

**Pull request merged**